draft-irtf-cfrg-vdaf.md

```diff
@@ -4967,31 +4967,39 @@ The aggregation parameter is encoded as follows:
 struct {
     uint16_t level;
     uint32_t num_prefixes;
-    opaque packed_prefixes[packed_len];
+    opaque encoded_prefixes[prefixes_len];
 } Poplar1AggParam;
 ~~~
 
 The fields in this struct are: `level`, the level of the IDPF tree of each
 prefixes; `num_prefixes`, the number of prefixes to evaluate; and
-`packed_prefixes`, the sequence of prefixes packed into a byte string of
-length `packed_len`. The prefixes are encoded with the following procedure:
+`encoded_prefixes`, the sequence of prefixes encoded into a byte string of
+length `prefixes_len`. The prefixes are encoded with the following procedure:
 
 ~~~ python
-packed = 0
-for (i, prefix) in enumerate(prefixes):
-    packed |= prefix << ((level + 1) * i)
-packed_len = ((level + 1) * len(prefixes) + 7) // 8
-packed_prefixes = to_be_bytes(packed, packed_len)
+prefixes_len = ((level + 1) + 7) // 8 * len(prefixes)
+encoded_prefixes = bytearray()
+for prefix in prefixes:
+    for chunk in itertools.batched(prefix, 8):
+        byte_out = 0
+        for bit_position, bit in enumerate(chunk):
```

Comment:
nit: When unpacking a tuple, elsewhere in the code we prefer to the keep the parentheses.
```suggestion
        for (bit_position, bit) in enumerate(chunk):
```